scripts/bench

```diff
@@ -15,7 +15,7 @@ echo "Writing to ${dest}"
 
 for branch in "${2}" "${1}"; do
   git checkout "${branch}"
-  make bench PKG="${PKG}" BENCHES="${BENCHES}" TESTFLAGS="-count 10" > "${dest}/bench.${branch}" 2> "${dest}/log.txt"
+  make bench PKG="${PKG}" BENCHES="${BENCHES}" TESTFLAGS="-count 10 -benchmem" > "${dest}/bench.${branch}" 2> "${dest}/log.txt"
```

Comment:
Would be great if you could split this into its own commit with some rationale. Only if you need go through CI again for something else, though.